lumigator/schemas/lumigator_schemas/workflows.py

```diff
@@ -3,13 +3,20 @@
 
 from pydantic import BaseModel
 
-from lumigator_schemas.jobs import JobResults, JobStatus
+from lumigator_schemas.jobs import JobResults, LowercaseEnum
 
 
-class WorkflowCreate(BaseModel):
+class WorkflowStatus(LowercaseEnum):
+    CREATED = "created"
+    RUNNING = "running"
+    FAILED = "failed"
+    SUCCEEDED = "succeeded"
+
+
+class WorkflowCreateRequest(BaseModel):
```

Comment:
I'd take the opportunity to put a list of JobCreates as a param. This avoids misusing params like "model", which is only necessary in the inference. In a similar way, the WorkflowResponse can carry a list of JobResponses (or a summary with only uuids and status, maybe).

Reply:
I agree with doing this in the long term, but for the moment this is consistent with the way we create workflows/(old experiments). We have an issue created to support declarative definitions for defining workflows https://github.com/mozilla-ai/lumigator/issues/774 and I'd like to have that issue be the one that updates the API to allow a client to specify the type of workflow configuration